core/controller/root.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * core/controller/root.js
+ * Handles the dashboard and static pages
+ *
+ * @exports {Object} default - Koa router
+ */
+
+import Router from 'koa-router'
```

Comment:
all the core files are linted. The only reason the appHook files were not is because I have yet to update them. There should be an open PR for that now